frontend/src/components/Dashboard.vue

```diff
@@ -71,4 +115,38 @@
     </v-row>
 
   </v-container>
-</template>
\ No newline at end of file
+</template>
+
+<script>
+import { mapState } from 'pinia';
+import { authStore } from '../store/modules/auth';
+import { useCodeStore } from '../store/modules/codeStore';
+import ApiService from '../common/apiService';
+
+export default {
+  data: () => ({
+    localreports: []
+  }),  
+  watch: {
+    reports(val) {
+      this.localreports = val;
+      //this.localreports.report.report_start_date.format('dddd MMMM D, YYYY');
+    }, 
+  },      
+  computed: {
+    ...mapState(authStore, ['userInfo']),
+    ...mapState(useCodeStore, ['naicsCodes', 'reports']),
```

Comment:
Don't fetch the reports from the codeStore.  Instead fetch them from ApiService.getReportsByStatus().